### PR TITLE
GitHub integration update

### PR DESCRIPTION
**Pull request by jmfontaine**
# Description of the change

Major update of the GitHub integration page.

Preview: https://spacelift-user-documentation-pr-59.onrender.com/integrations/source-control/github.html

## Checklist

Please make sure that the proposed change checks all the boxes below before requesting a review:

- [x] I have reviewed the [guidelines for contributing](https://github.com/spacelift-io/user-documentation/blob/main/CONTRIBUTING.md) to this repository.
- [x] The preview looks fine.
- [x] The tests pass.
- [x] The commit history is clean and meaningful.
- [x] The pull request is opened against the `main` branch.
- [x] The pull request is no longer marked as a draft.
- [x] You agree to license your contribution under the [MIT license](https://github.com/spacelift-io/user-documentation/blob/main/LICENSE) to Spacelift.

If the proposed change is ready to be merged, please request a review from `@spacelift-io/solutions-engineering`. Someone will review and merge the pull request.

Thank you for your contribution! 🙇

### CODE REVIEW

File: docs/integrations/source-control/github.md [hunk 1]
```diff
@@ -6,86 +6,131 @@ description: >-
...
+### Using the Marketplace application (Preferred)
 
-![](<../../assets/screenshots/Screen Shot 2022-04-20 at 4.25.49 PM.png>)
+The easiest way to connect your GitHub account (personal or organization) is to install the [Spacelift application](https://github.com/marketplace/spacelift-io){: rel="nofollow"} from the GitHub Marketplace.
 
-This should open a form like this one:
+![](<../../assets/screenshots/CleanShot 2022-09-15 at 17.27.12.png>)
 
-![](<../../assets/screenshots/Screen Shot 2022-04-20 at 4.28.27 PM.png>)
+At the bottom of the page, select the GitHub account where to install the Spacelift application and then click on the _Install it for free_ button.
```
Comment (adamconnelly) [inline docs/integrations/source-control/github.md:21]:
```suggestion
At the bottom of the page, select the GitHub account where you want to install the Spacelift application and then click on the _Install it for free_ button.
```

File: docs/integrations/source-control/github.md [hunk 2]
```diff
@@ -6,86 +6,131 @@ description: >-
...
+![](<../../assets/screenshots/CleanShot 2022-09-15 at 17.44.49.png>)
 
-Before you can complete this step you need to create a GitHub App within GitHub. Start by navigating to the _GitHub Apps_ page in the _Developer Settings_ for your account/organization, and clicking on _New GitHub App._
+On the next page click on the _Complete order and begin installation_ button.
 
-You will need the Webhook endpoint and Webhook secret while creating your App, so take a note of them.
+![](<../../assets/screenshots/CleanShot 2022-09-15 at 17.46.56.png>)
 
-You can either create the App in an individual user account or within an organization account_:_
+On the last screen, you will be able to select which repositories should be available to Spacelift, review the required permissions. When ready, click on the _Install_ button, and voilà!
```
Comment (adamconnelly) [inline docs/integrations/source-control/github.md:29]:
```suggestion
On the last screen, you will be able to select which repositories should be available to Spacelift, and review the required permissions. When ready, click on the _Install_ button, and voilà!
```

File: docs/integrations/source-control/github.md [hunk 3]
```diff
@@ -6,86 +6,131 @@ description: >-
...
 
-Click on the Save button to save your integration settings.
+Congratulations! You are almost done! 🎉
 
-### Installing your GitHub App
+The last step is to install the application you just created so that Spacelift can interact with GitHub. This is what the next section is about.
+
+### Installing the GitHub custom application
 
 Now that you've created a GitHub App and configured it in Spacelift, the last step is to install your App in one or more accounts in your GitHub account. To do this, go back to GitHub, find your App in the GitHub Apps page in your account settings, and click on the _Edit_ button next to it:
```
Comment (adamconnelly) [inline docs/integrations/source-control/github.md:135]:
```suggestion
Now that you've created a GitHub App and configured it in Spacelift, the last step is to install your App in one or more accounts or organizations you have access to. To do this, go back to GitHub, find your App in the GitHub Apps page in your account settings, and click on the _Edit_ button next to it:
```

Sorry Jean-Marc - I know this is existing copy, I just noticed it while proofing.

File: docs/integrations/source-control/github.md [hunk 4]
```diff
@@ -6,86 +6,131 @@ description: >-
...
 
-![](<../../assets/screenshots/image (55).png>)
+    You can create the custom Spacelift application for GitHub manually.
 
-Once your App has been created, make a note of the _App ID_ in the _About_ section:
+    !!! warning
+        This should be used as a last resort when the other methods can not be used as it is more tedious and error-prone.
 
-![](<../../assets/screenshots/image (56).png>)
+    This should open a form like this one:
```
Comment (adamconnelly) [inline docs/integrations/source-control/github.md:60]:
```suggestion
    After selecting the option to enter your details manually, you should see the following form:
```

File: docs/integrations/source-control/github.md [hunk 5]
```diff
@@ -274,3 +311,37 @@ Whenever a review is added or dismissed from a Pull Request, we check whether a
...
+
+    Go to the _Developer settings_ of the GitHub account, then in the _GitHub Apps_ section click on the _Edit_ button for the Spacelift application.
+
+    ![](<../../assets/screenshots/CleanShot 2022-09-16 at 10.28.11.png>)
+
+    On the page for the Spacelift application, go to the *Advanced* section and click on the _Delete GitHub App_ button. Confirm by typing the name of the application and it is gone.
+
+    ![](<../../assets/screenshots/CleanShot 2022-09-16 at 10.24.08.png>)
+
+    If you no longer need the integration, you can remove it via the *Unlink* button on the *Source Code* page:
```
Comment (adamconnelly) [inline docs/integrations/source-control/github.md:345]: You might be as well just telling users to remove the integration. At the moment there's no way to edit an existing integration (via the frontend), so as soon as they've deleted the app the integration is useless.

**Review by adamconnelly — APPROVED** [review]
Looks great - just left some relatively minor comments.

File: docs/integrations/source-control/github.md (hunk 3 above)
Comment (jmfontaine) [reply docs/integrations/source-control/github.md:135]: No worries. Let's fix old and new issues. 😄

File: docs/integrations/source-control/github.md (hunk 5 above)
Comment (jmfontaine) [reply docs/integrations/source-control/github.md:345]: Okay. I will rephrase this.

Comment (jmfontaine) [reply docs/integrations/source-control/github.md]: Done

**Review by adamconnelly — APPROVED** [review]

**Pull request merged**